src/main/java/at/hannibal2/skyhanni/features/rift/everywhere/UbikReminder.kt

```diff
@@ -0,0 +1,49 @@
+package at.hannibal2.skyhanni.features.rift.everywhere
+
+import at.hannibal2.skyhanni.api.event.HandleEvent
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.events.chat.SkyHanniChatEvent
+import at.hannibal2.skyhanni.features.rift.RiftApi
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.DelayedRun
+import at.hannibal2.skyhanni.utils.RegexUtils.matches
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import kotlin.time.Duration.Companion.hours
+
+@SkyHanniModule
+object UbikReminder {
+
+    private val config get() = RiftApi.config.area.mountaintop
+
+    private var isTimerRunning = false
+    private val patternGroup = RepoPattern.group("rift.ubik")
+
+    /**
+     * REGEX-TEST: ROUND 2 (FINAL):
+     */
+    private val ubikRoundPattern by patternGroup.pattern(
+        "reminder",
+        "ROUND [1-9] \\(FINAL\\):",
+    )
+
+    @HandleEvent(onlyOnIsland = IslandType.THE_RIFT)
+    fun onChat(event: SkyHanniChatEvent) {
+        if (!config.ubikReminder) return
+        val message = event.message
+        if (ubikRoundPattern.matches(message) && !isTimerRunning) {
```

Comment:
```suggestion
        if (ubikRoundPattern.matches(event.message) && !isTimerRunning) {
```

not needed to make a variable